modules/administration_manual/pages/maintenance/encryption/migrating-from-user-key-to-master-key.adoc

```diff
@@ -1,34 +1,36 @@
-= Migrating User Key Encryption to Master Key Encryption
+= Migrating User Key-based Encryption to Master Key Encryption
 
-== Why Should I Move Away From User Key-based Encryption
+== Why Should I Move Away From User Key-based Encryption?
 
-NOTE: User key-based encryption is planned to be removed from ownCloud in the near future. For new installations, Master key-based encryption is the recommended setup. The decryption workflow described here will only work when users have enabled password recovery and if an admin recovery password is available for the ownCloud administrator.
+NOTE: User key-based encryption is planned to be removed from ownCloud in the near future. While it is a bit more secure than a central encryption approach, User key-based encryption has some disadvantages. It blocks some additional functions such as the integration of an online editor like LibreOffice or OnlyOffice into ownCloud and can cause problems when sharing files with groups. Therefore Master-key-based encryption is now the recommended setup fr all new insallations.
```

Comment:
Please split each sentence onto a separate line. Secondly, in the last sentence, it should end with "recommended setup for all new installations".

Reply:
A typo in the last sentence "fr all new insallations." the `for`

Reply:
Spotted that one and left a comment already.